### DISCUSSION TITLE
Welcome to Opore-Mod Discussions!

### DISCUSSION THREAD

**Discussion by xamionex** [xamionex/Opore-Mod#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Joachim-barre** [comment]
I want to devlopp mod and play

**Reply by xamionex** [reply]
I can't make you a maintainer for 10 days because I'm on a trip, you'll have to wait
